package.json

```diff
@@ -1,76 +1,31 @@
 {
   "name": "sketch-api",
   "private": true,
-  "version": "2.0.0",
+  "version": "0.1.0",
```

Comment:
I know it doesn't really matter since it doesn't get published to npm. Is it even necessary/required to be included?

Reply:
I think commands like `npm install` fail if `version` isn't set to _some_ valid semver. The default value for `version` in a fresh package.json created by npm is `1.0.0`. I'll set it to that, since I think it's the least surprising/misdirecting for a project that is technically unversioned.

Reply:
That's a change that could impact plugins: the API version is exposed in the API. It was 2.0.0 because there was an API 1.0.

Reply:
Ah I see 👍